settingtypes.txt

```diff
@@ -46,3 +46,12 @@ share_bones_time (Bone share time) int 1200 0
 
 #    Replaces old stairs with new ones. Only required for older worlds.
 enable_stairs_replace_abm (Replace old stairs) bool false
+
+
+#    If enabled, the reason why bones where dropped or not dropped, along
+#    with death location, is written to the minetest log.
+enable_bones_logging (Log location of deaths and reason if no bones) bool true
+
+#    If enabled, the reason why bones where dropped or not dropped, along
```

Comment:
Possible rephrasing here too: `Sends the player a message about the condition and location of their bones on death.`